web/stories/VideoEmbed.stories.tsx

```diff
@@ -12,17 +12,20 @@ const Template = ({
   title: string;
   width: number;
   height: number;
-}) => (
-  <iframe
-    src={`${origin}/embed/video?${query}`}
-    title={title}
-    height={`${height}px`}
-    width={`${width}px`}
-    referrerPolicy="origin"
-    scrolling="no"
-    allowFullScreen
-  />
-);
+}) => {
+  const queryString = query ? `?${query}` : '';
+  return (
+    <iframe
+      src={`${origin}/embed/video${queryString}`}
```

Comment:
The problem here is the origin isn't directly connected with the state of the Storybook story. Because an instance of Storybook isn't running Owncast it can't point to itself, so I'm unfortunately not sure how to make a story for the embed that's not more than a blank iframe.